src/EditorFeatures/CSharpTest/InitializeParameter/InitializeMemberFromParameterTests.cs

```diff
@@ -648,5 +650,164 @@ public C(string s, int i)
     }
 }");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInitializeParameter)]
+        public async Task TestInitializePropertyWithRequiredAccessibilityOmitIfDefault()
+        {
+            await TestInRegularAndScript1Async(
+@"
+class C
+{
+    readonly int test = 5;
+
+    public C(int test, int [|test2|])
+    {
+    }
+}",
+@"
+class C
+{
+    readonly int test = 5;
+
+    public C(int test, int test2)
+    {
+        Test2 = test2;
+    }
+
+    int Test2 { get; }
+}", index: 0, parameters: new TestParameters(options: Option(CodeStyleOptions.RequireAccessibilityModifiers, AccessibilityModifiersRequired.OmitIfDefault, NotificationOption.Warning)));
```

Comment:
can you make some static members for these options?  it's easier to read the test if it says: `parameters: OmitIfDefault_Warning`

Reply:
This is wrong.  You should be making a public property.  but this will make a private property.